cetmix_tower_server/tests/common.py

```diff
@@ -208,6 +208,30 @@ def setUp(self, *args, **kwargs):
             }
         )
 
+        # Command
+        self.command_run_flight_plan = self.Command.create(
+            {
+                "name": "Run Flight Plan",
+                "action": "plan",
+                "flight_plan_id": self.plan_1.id,
+            }
+        )
+
+        # Flight plan
+        self.plan_2 = self.Plan.create(
```

Comment:
Same here. We are using this in the `test_plan.py' only.  Probably it would be reasonable to move it to the `setUp` of the corresponding class.
Naming should be more relevant as well to improve readability. Eg `parent_flight_plan`